provisioning/device/src/PlugAndPlayConvention/PnpConventionHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Provisioning.Client.Extensions;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.PlugAndPlayConvention
+{
+    /// <summary>
+    /// A helper class for formatting the data as per plug and play convention.
+    /// </summary>
+    public static class PnpConventionHelper
+    {
+        /// <summary>
+        /// Create the DPS payload to provision a device as plug and play.
+        /// For more information, see https://docs.microsoft.com/en-us/azure/iot-pnp/howto-certify-device.
```

Comment:
For more information on device provisioning service and plug and play compatibility, and PnP device certification, see...